.github/workflows/testing.yml

```diff
@@ -19,3 +19,30 @@ jobs:
         with-submodules: true
     - name: Build
       run: docker build .
+
+  flake8:
+    name: Flake8
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Flake8
+      uses: TrueBrain/actions-flake8@master
+      with:
+        path: fetch_downloads
+
+  black:
+    name: Black
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
```

Comment:
3.8? Or do we want to maintain compatibility with older versions? (Not a bad thing, just querying)